tooling/noir_js/src/serialize.ts

```diff
@@ -0,0 +1,17 @@
+import { WitnessMap, compressWitness } from '@noir-lang/acvm_js';
+import { decompressSync as gunzip } from 'fflate';
+import { base64Decode } from './base64_decode.js';
+
+// After solving the witness, to pass it a backend, we need to serialize it to a Uint8Array
+export function witnessMapToUint8Array(solvedWitness: WitnessMap): Uint8Array {
+  // TODO: We just want to serialize, but this will zip up the witness
+  // TODO so its not ideal
```

Comment:
TODO here, is there an issue?

Reply:
https://github.com/noir-lang/noir/issues/2771